.github/workflows/build.yml

```diff
@@ -54,12 +54,12 @@ jobs:
       - name: Update Package Lists
         run: sudo apt-get update
       - name: Install Dependencies
-        run: sudo apt-get install valgrind g++ make --fix-missing
+        run: sudo apt-get install valgrind g++ make gcc-avr avr-libc binutils-avr --fix-missing

```

Comment:
should this just be

     make install
     
and use the makefile target for installing dependencies?  That allows people to run it themselvse with make, where as this can only be run if you have some tool to run the yml jobs -- and only this job.  Not nearly as easy.